lib/recipe/widgets/recipe_screen.dart

```diff
@@ -188,8 +187,52 @@ class Ingredients extends StatelessWidget {
                               Navigator.pushNamed(
                                   context, PageRouter.ingredient,
                                   arguments: [recipeIndex, index])
+                            },
+                        onLongPress: () => {
+                              showDialog<String>(
+                                  context: context,
+                                  builder: (BuildContext context) =>
+                                      DeleteIngredient(
+                                          recipe: state.recipes[recipeIndex],
+                                          ingredient: state.recipes[recipeIndex]
+                                              .ingredients[index]))
```

Comment:
This works nicely. Again, we just have to find out if it is preferred to have delete buttons or long-presses.